public/javascripts/main.js

```diff
@@ -143,14 +143,14 @@ function populateTypeahead(element, source, displayFun, suggestionFun, updateVal
     element.typeahead({
         hint: true,
         highlight: true,
-        minLength: 0
+        minLength: 2
```

Comment:
Just to confirm... If nothing is typed a (scrollable) list of 100 entries is displayed and selectable. This behavior is essential if someone doesn't know what he/she is looking for.

Reply:
No, and it should not. If that is the current way of exploring fields in Ada, then it needs to be re-designed. A text field is not for data exploration.

Reply:
As we discussed it's completely fine to allow this change for fields (show any results only when 2 chars are entered) but typeaheads are used for many different scenarios (e.g. view, filter, ML classifier selection). For the non-field searches we want to display all the options to choose from (scrollable), i.e., typeahead works there essentially as an intelligent select box.

To allow both use cases to be served probably the easiest option would be to expose the minLength option.

Reply:
I totally forgot about that. Of course you are right. I only had the "5000 fields" example in mind, but for views and filters this is perfectly fine.

Let me fix it real quick